three.c

```diff
@@ -0,0 +1,63 @@
+/* This program tells whether 3 values form a triangle, what type of triangle these form according to sides, and it handles invalid input. */
+#include <stdio.h>
```

Comment:
Very good job commenting and getting the functions written for the most part. You didn't handle floats well enough (it didn't correctly prompt for new input) and invalid triangles weren't correctly handled - they always evaluated as scalenes when that shouldn't have been the case.